### PR TITLE
Remove any usage of bare exception.

### PR DESCRIPTION
**Pull request by MaxJPRey**
It's generally considered good practice to use `except Exception as e`: rather than a bare `except:` because it avoids catching some exceptions that you typically don't want to catch, like keyboard interrupts and system exits. It also allows you to access and use the actual exception object, which can be helpful for debugging.

### CODE REVIEW

**Review by SMoraisAnsys — DISMISSED** [review]

**Comment by MaxJPRey** [comment]
> @MaxJPRey To avoid codacy failure, I would recommand to not use `except Exception as e:` but `except Exception:` since `e` is never used afterward.

@SMoraisAnsys I fully agree. I will do it.

**Comment by SMoraisAnsys** [comment]
@MaxJPRey Since you are busy, feel free to let me do the changes. Just ping me so that I get notified 😄

**Comment by MaxJPRey** [comment]
> @MaxJPRey Since you are busy, feel free to let me do the changes. Just ping me so that I get notified 😄

@SMoraisAnsys It is done.

File: _unittest/test_01_Design.py [hunk 1]
```diff
@@ -374,15 +374,15 @@ def test_36_test_load(self, add_app):
             f.write(" ")
         try:
             hfss = Hfss(projectname=file_name2, specified_version=desktop_version)
-        except:
+        except Exception as e:
```
Comment (SMoraisAnsys) [inline _unittest/test_01_Design.py:377]:
```suggestion
        except Exception:
```

File: _unittest/test_01_Design.py [hunk 2]
```diff
@@ -374,15 +374,15 @@ def test_36_test_load(self, add_app):
...
             assert True
         try:
             os.makedirs(os.path.join(self.local_scratch.path, "test_36_2.aedb"))
             file_name3 = os.path.join(self.local_scratch.path, "test_36_2.aedb", "edb.def")
             with open(file_name3, "w") as f:
                 f.write(" ")
             hfss = Hfss3dLayout(projectname=file_name3, specified_version=desktop_version)
-        except:
+        except Exception as e:
```
Comment (SMoraisAnsys) [inline _unittest/test_01_Design.py:385]:
```suggestion
        except Exception:
```

**Review by SMoraisAnsys — CHANGES_REQUESTED** [review]
Found 2 `except Exception as e` left. Otherwise, all good for me :)
Thanks for the huge refactoring

**Review by SMoraisAnsys — DISMISSED** [review]

**Review by Samuelopez-ansys — APPROVED** [review]

File: pyaedt/aedt_logger.py [hunk 3]
```diff
@@ -590,7 +590,8 @@ def _log_on_dekstop(self, message_type, message_text, level=None, proj_name=None
                 des_name = des_name[des_name.find(";") + 1 :]
             try:
                 self._desktop.AddMessage(proj_name, des_name, message_type, message_text)
-            except:
+
+            except Exception:
```
Comment (Samuelopez-ansys) [inline pyaedt/aedt_logger.py:594]:
```suggestion
            except Exception:  # pragma: no cover
```

File: pyaedt/aedt_logger.py [hunk 4]
```diff
@@ -658,7 +659,7 @@ def clear_messages(self, proj_name=None, des_name=None, level=2):
             des_name = self.design_name
         try:
             self._desktop.ClearMessages(proj_name, des_name, level)
-        except:
+        except Exception:
```
Comment (Samuelopez-ansys) [inline pyaedt/aedt_logger.py:662]:
```suggestion
        except Exception:  # pragma: no cover
```

**Review by Samuelopez-ansys — APPROVED** [review]

**Review by Samuelopez-ansys — DISMISSED** [review]

**Comment by Samuelopez-ansys** [comment]
@MaxJPRey I cancelled the workflow before, because there are some unit test failing, apart from Codacy that is complaining. I will debug next week and try to find the issue.

**Review by SMoraisAnsys — APPROVED** [review]
LGTM

**Pull request merged**